### PR TITLE
Model: AdaBoost

### PR DESCRIPTION
**Pull request by buesst1**

### CODE REVIEW

**Comment by ghost** [comment]
# RUN: AdaBoostStumpClassifier, 2023-06-15 14-41-16

## Hyperparameters

| Path        | Param                                  | main                           | workspace               |
|-------------|----------------------------------------|--------------------------------|-------------------------|
| params.yaml | dvclive.model                          | HistGradientBoostingClassifier | AdaBoostStumpClassifier |
| params.yaml | dvclive.model_hparams.learning_rate    | 0.2                            | 0.5                     |
| params.yaml | dvclive.model_hparams.max_bins         | 255                            | -                       |
| params.yaml | dvclive.model_hparams.max_iter         | 2000                           | -                       |
| params.yaml | dvclive.model_hparams.max_leaf_nodes   | 31                             | -                       |
| params.yaml | dvclive.model_hparams.min_samples_leaf | 20                             | -                       |
| params.yaml | dvclive.model_hparams.n_estimators     | -                              | 200                     |
| params.yaml | dvclive.model_hparams.random_state     | -                              | 42                      |

## Metrics

| Path                 | Metric          | main    | workspace   | Change   |
|----------------------|-----------------|---------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc  | 0.98289 | 0.5242      | -0.45869 |
| dvclive/metrics.json | test.epoch.f1   | 0.98085 | 0.46119     | -0.51965 |
| dvclive/metrics.json | train.epoch.acc | 1.0     | 0.54989     | -0.45011 |
| dvclive/metrics.json | train.epoch.f1  | 1.0     | 0.49789     | -0.50211 |

## Plots

### Confusion Matrix

Confusion Matrix

### CV Metrics

CV Metrics

**Comment by gabrieltorresgamez** [comment]
Don't merge!!!

**Comment by ghost** [comment]
# RUN: AdaBoostStumpClassifier, 2023-06-16 09-08-34

## Hyperparameters

| Path        | Param                                  | main                           | workspace               |
|-------------|----------------------------------------|--------------------------------|-------------------------|
| params.yaml | dvclive.model                          | HistGradientBoostingClassifier | AdaBoostStumpClassifier |
| params.yaml | dvclive.model_hparams.learning_rate    | 0.2                            | 0.5                     |
| params.yaml | dvclive.model_hparams.max_bins         | 255                            | -                       |
| params.yaml | dvclive.model_hparams.max_iter         | 2000                           | -                       |
| params.yaml | dvclive.model_hparams.max_leaf_nodes   | 31                             | -                       |
| params.yaml | dvclive.model_hparams.min_samples_leaf | 20                             | -                       |
| params.yaml | dvclive.model_hparams.n_estimators     | -                              | 200                     |
| params.yaml | dvclive.model_hparams.random_state     | -                              | 42                      |
| params.yaml | dvclive.seed                           | 42                             | 43                      |

## Metrics

| Path                 | Metric          | main    | workspace   | Change   |
|----------------------|-----------------|---------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc  | 0.98289 | 0.55778     | -0.42511 |
| dvclive/metrics.json | test.epoch.f1   | 0.98085 | 0.52321     | -0.45764 |
| dvclive/metrics.json | train.epoch.acc | 1.0     | 0.58012     | -0.41988 |
| dvclive/metrics.json | train.epoch.f1  | 1.0     | 0.51683     | -0.48317 |

## Plots

### Confusion Matrix

Confusion Matrix

### CV Metrics

CV Metrics
